modules/shared/components/molecules/UploadingImage/UploadingImage.test.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
```

Comment:
I don't just a snapshot is sufficient for this component. you need to check that the click handler and text input interactions working as expected

Reply:
i agree with you, my mistake.